erb_to_haml

tsutaya100507 opened the pull request:
# WHAT
erbファイルをHamlに変換する。

# WHY
Hamlの記法でビューファイルを作成可とするため。

Commit:
create pull request

Commit:
erb_to_haml

Erbファイルをhamlに変換

takuya032 commented on Gemfile:
```diff
@@ -51,3 +51,5 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+# Haml
+gem 'erb2haml'
```
このgemはローカルでしか必要ないので、使用環境を絞ってください！

tsutaya100507 replied:
修正しました！

takuya032 commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,10 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットは使わないでくださいね。

tsutaya100507 replied:
修正しました！

takuya032 commented on app/views/layouts/mailer.html.haml:
```diff
@@ -0,0 +1,8 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=utf-8", "http-equiv" => "Content-Type"}/
```
こちらもですね

tsutaya100507 replied:
修正しました！

ghost commented:
今はローカルでviewが表示されてますでしょうか？
みたところhaml-railsが入ってなさそうなので、viewの変換ができていないと思います。

Commit:
edit_Gemfile

Commit:
edit_http.haml

application.html.haml及びmailer.html.hamlのハッシュロケットをシンボル型に書き換え。

※ハッシュロケットは古い書き方であり、カリキュラム上も使用しない取り決めとなっていることから要注意。
また、シンボル型のキーにはハイフンが使えないため記法が難しくなるのでこれについても注意。
